shared/controller-api/src/main/proto/Controller.proto

```diff
@@ -154,7 +154,13 @@ message ContinuationToken {
  message StreamsInScopeResponse {
     repeated StreamInfo streams = 1;
     ContinuationToken continuationToken = 2;
-}
+    enum Status {
+        SUCCESS = 0;
+        FAILURE = 1;
+        SCOPE_NOT_FOUND = 2;
```

Comment:
Instead of creating another return value, could we keep it at success/failure and include the failure message the reason? The advantage of throwing a explicit exception is that the application can react to it and create the scope. Is it what we are trying to achieve?

Reply:
Since we are throwing explicit new exception so having a specific error code rather than a message is better